src/views/SettingsView.vue

```diff
@@ -71,6 +79,10 @@ const openTradesOptions = [
   { value: OpenTradeVizOptions.asTitle, text: 'Show in title' },
   { value: OpenTradeVizOptions.noOpenTrades, text: "Don't show open trades in header" },
 ];
+const colorPreferenceOptions = [
+  { value: 'greenUp', text: 'Green Up/Red Down' },
+  { value: 'redUp', text: 'Green Down/Red Up' },
+];
```

Comment:
I don't really like this way to set this up - as it's pretty restrictive, and will only fill exactly one requirement, ignoring all other accessibility requirements.

a better way is to allow the user to select custom colors (a user may be red/green colorblind, so swapping up with down won't help that case).
We can still have a "swap" button for candles (will ONLY apply to candles) - but the actual color should be selectable independently.